logging/google/cloud/logging/logger.py

```diff
@@ -452,7 +455,8 @@ def commit(self, client=None):
             kwargs['labels'] = self.logger.labels
 
         entries = []
-        for entry_type, entry, labels, iid, severity, http_req, timestamp in self.entries:
+        for entry_type, entry, labels, iid, severity, http_req, timestamp in \
+                self.entries:
```

Comment:
I hate to nitpick, but we really try hard to avoid backslash continuations.  Can you wrap the unpack-variables in parentheses instead?  E.g:

```python
    for (entry_type, entry, labels, iid, severity, http_req,
         timestamp) in self.entries:
```